README.Rmd

```diff
@@ -159,3 +159,39 @@ By contributing to this project, you agree to abide by its terms.
 ```{r message=FALSE, warning=FALSE}
 citation("simulist")
 ```
+
+## Related projects
+
+This project has some overlap with other R packages. Here we list these packages and provide a table of features and attributes that are present for each package to help decide which package is appropriate for each use-case.
+
+In some cases the packages are dedicated to simulating line list and other epidemiological data (e.g. {simulist}), in others the line list simulation is one part of a wider R package (e.g. {EpiNow}). 
+
+- [`{LLsim}`](https://github.com/jrcpulliam/LLsim) simulates line list data using a stochastic SIR model with a fixed population with observation and reporting delays.
+- [`{simulacr}`](https://github.com/reconhub/simulacr) uses a branching process model to simulate cases and contacts for an outbreak. It simulates transmission of infections using other epidemiological R packages (`{epicontacts}` and `{distcrete}`) to parameterise and plot simulated data.
+- [`{epidict}`](https://github.com/R4EPI/epidict) is a package for data dictionaries for outbreaks and surveys. It can generate line list data using an outbreak data dictionary (`gen_data()`). Currently MSF outbreak data dictionaries for Acute Jaundice Syndrome, Cholera, Measles and Meningitis are available.
+- [`{EpiNow}`](https://github.com/epiforecasts/EpiNow) - a now deprecated R package - includes the `simulate_cases()` and `generate_pseudo_linelist()` functions for generating line list data.
+- [generative-nowcasting](https://github.com/adrian-lison/generative-nowcasting) is a set of R scripts and functions to perform epidemiological nowcasting. There are [functions to simulate line list data](https://github.com/adrian-lison/generative-nowcasting/blob/bf48e027e82ce9d42de468d0b708d010253b7475/code/utils/utils_simulate.R) within the repository, but the repository is not (and does not contain) an R package. Functions can be sourced. Cases are simulated with a renewal process and the simulation can incorporate epidemiological delays and ascertainment.
+
+<details> 
+
+<summary> Table of line list simulator features </summary>
+
+
+|                | {simulist}     | {LLsim}        | {simulacr}     | {epidict}      | {EpiNow}       | generative-nowcasting       | 
+| -------------- | -------------- | -------------- | -------------- | -------------- | -------------- | -------------- |
+| Simulates line list | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Simulates contacts | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :x: | :x: |
+| Parameterised with epi distributions | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :white_check_mark: | :white_check_mark: |
+| Interoperable with {epicontacts} | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :x: | :x: |
```

Comment:
Columns that I think need a footnote: parameterised with epi distributions, explicit population size, unit testing

Reply:
I've added footnotes for all of these fields in 2ff93f0e16b283d30ff09a19c5436516247bcc7d.